dataproxy/vercel-cli-serverless-functions/run.sh

```diff
@@ -22,17 +22,9 @@ echo "Deployed to ${DEPLOYED_URL}"
 
 sleep 15
 
-OUTPUT=$(yarn -s vercel logs $DEPLOYED_URL --token=$VERCEL_TOKEN --scope=$VERCEL_ORG_ID)
+OUTPUT=$(pnpm vercel logs $DEPLOYED_URL --token=$VERCEL_TOKEN --scope=$VERCEL_ORG_ID)
 echo "${OUTPUT}"
 
-# Check the Vercel Build Logs for the postinstal hook"
-if echo "${OUTPUT}" | grep -q 'prisma generate || true'; then
-  echo 'Postinstall hook was added'
-else
-  echo "Postinstall hook was NOT ADDED"
-  exit 1
-fi
-
```

Comment:
unrelated

Reply:
It does not work in some cases with pnpm it seems